wikipron/languagecodes.py

```diff
@@ -252,10 +252,88 @@
     "tetelcingo nahuatl": "Tetelcingo Nahuatl",
     # Bouyei. ISO 639-3 only.
     "pcc": "Bouyei",
+    "bouyei": "Bouyei",
     # Lamboya. ISO 639-3 only.
-    "lmy": "Lamboya",
+    "lmy": "Laboya",
+    "laboya": "Laboya",
     # Moroccan Arabic. ISO 639-3 only.
     "ary": "Moroccan Arabic",
+    "moroccan arabic": "Moroccan Arabic",
     # Mandarin Chinese. ISO 639-3 only.
-    "cmn": "Chinese"
+    "cmn": "Chinese",
+    "chinese": "Chinese",
+    # Abkhaz. Would be Abkhazian in ISO 639.
+    "abk": "Abkhaz",
+    "abkhaz": "Abkhaz",
+    # Avar. Would be Avaric in ISO 639.
+    "ava": "Avar",
+    "avar": "Avar",
+    # Buryat. Would be Buriat in ISO 639.
+    "bua": "Buryat",
+    "buryat": "Buryat",
+    # Chukchi. Would be Chukot in ISO 639.
+    "ckt": "Chukchi",
+    "chukchi": "Chukchi",
+    # Burushaski. ISO 639-3 only.
+    "bsk": "Burushaski",
+    "burushaski": "Burushaski",
+    # Evenki. Not in iso639 lib.
+    "evn": "Evenki",
+    "evenki": "Evenki",
+    # Southern Yukaghir. Not in iso639 lib.
+    "yux": "Southern Yukaghir",
+    "southern yukaghir": "Southern Yukaghir",
+    # Tundra Nenets. Not in iso639 lib.
+    "yrk": "Tundra Nenets",
+    "tundra nenets": "Tundra Nenets",
+    # Estonian. Not in iso639 lib.
+    "ekk": "Estonian",
```

Comment:
`ekk` looks fine to me here. The "iso639" package can already resolve `est` as Estonian, probably because `est` is in ISO 639-2 as well. `ekk` is ISO 639-3 only -- thanks for adding it!

It seems fine to point both `ekk` and `est` to "Estonian" for our purposes. I haven't found [Estonian entries on Wiktionary](https://en.wiktionary.org/wiki/Category:Estonian_terms_with_IPA_pronunciation) that would have a dialect label for "Standard Estonian" or something. If we learn more about Estonian (broadly construed) in the future and we decide to update the mapping, we'll certainly do so.